idris-prover.el

```diff
@@ -218,7 +218,7 @@ left margin."
           (let ((tactic-cmd (replace-regexp-in-string
                              "\\`[ \t\n]*" ""
                              (replace-regexp-in-string "
" " " tactic-text))))
-            (idris-rex () (list ':interpret tactic-cmd)
+            (idris-rex () (list ':interpret tactic-cmd) nil
```

Comment:
I don't remember actually changing that line - my guess is that it's a Git hiccup related to the `^M` literal, which I can replace with something more readable. The reason for two replace-regexp-in-string calls is that the replacement string is different. The first removes leading whitespace, the second fixes up odd Windows newlines in parsed tactics, which the underlying Idris prover doesn't understand.